### PR TITLE
Include rewrite-tag-filter plugin for fluentd v0.12

### PR DESCRIPTION
**Pull request by naiduarvind**
Added fluent-plugin-rewrite-tag-filter to allow tags to be rewritten.
Issue: #67 

Signed-off-by: Arvind Naidu arvind@evozi.com

### CODE REVIEW

**Comment by naiduarvind** [comment]
@repeatedly, if you could have a look at this and have it out soonest. A blocker for my team as we are pushing for production by end of the month.

**Comment by repeatedly** [comment]
Please regenerate all image files with `make src-all`

**Comment by repeatedly** [comment]
@naiduarvind ping

**Comment by naiduarvind** [comment]
Signed-off-by: Arvind Naidu arvind@evozi.com

File: templates/Gemfile.erb [hunk 1]
```diff
@@ -12,6 +12,7 @@ gem "oj", "3.5.1"
 <% if !is_v1 %>
 gem "fluent-plugin-secure-forward"
 gem "fluent-plugin-record-reformer"
+gem "fluent-plugin-rewrite-tag-filter", "1.6.0"
```
Comment (errm) [inline templates/Gemfile.erb]: Some questions?
a) You shouldn't need to specify a version number here ... the dependencies are specified correctly on the gem so bundler will resolve the latest compatible version for you?
b) why are you not including the plugin for the v1 based images?

**Comment by repeatedly** [comment]
Please put signed-off-by in commit, not comment.

**Comment by jeffdesc** [comment]
Someone has an ETA on this feature? I could really use it.

**Comment by repeatedly** [comment]
I'm not sure because no response from patch author.

File: templates/Gemfile.erb (hunk 1 above)
Comment (naiduarvind) [reply templates/Gemfile.erb]: a) v1 and v0.12 uses a different version of the rewrite-plugin-filter - https://github.com/fluent/fluent-plugin-rewrite-tag-filter
b) Including them now

**Comment by naiduarvind** [comment]
Fixed the sign-off-by issue but I am still facing it. Can I get some pointers as to why it is not working?

cc: @repeatedly @errm

**Comment by errm** [comment]
There are 4 commits in this PR... you correctly signed 2 off them off 2 of them are not signed ...

I would recommend squashing all of your work down to 1 commit and ensuring it is signed...

I would do an interactive rebase for this:

`git rebase -i HEAD~4`

this will open your editor where you can reorder and mark your commits to squash or fixup ...

**Comment by naiduarvind** [comment]
I will decline this PR and create a new one as something is not working as expected. My apologies!

**Pull request closed without merging**